Dockerfile

```
@@ -1,23 +1,36 @@
-ENV PORT 8080
-ENV PYTHONPATH /src
+ENV LEVEL=INFO
+ENV SERVER=netius
+ENV SERVER_ENCODING=gzip
+ENV HOST=0.0.0.0
+ENV PORT=8080
+ENV PYTHONPATH=/src
 
-ADD requirements.txt /
-ADD extra.txt /
 ADD src /src
```

Hadolint (2.14.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)